src/GraphQL/GraphQL.csproj

```diff
@@ -6,6 +6,7 @@
     <Authors>Joe McBride</Authors>
     <Copyright>Copyright (c) 2015-2021 Joseph T. McBride, Ivan Maximov, Shane Krueger, et al. All rights reserved.</Copyright>
     <PackageTags>GraphQL;json;api</PackageTags>
+    <Nullable>enable</Nullable>
```

Comment:
Worth to move into Directory.Build.props

Reply:
Still not applied to every project 100%. It should be. Probably not much work left

Reply:
Can be disabled for tests.